### ISSUE TITLE
mixer type menu doesn't list 'Hardware' option

### ISSUE THREAD

**Issue by xipmix** [volumio/Volumio2-UI#707]
This may be intended behaviour, but it seems strange.
Cog->PlaybackOptions has the 'Mixer Type' option.

This seems to have defaulted to 'Hardware'
![mt1](https://user-images.githubusercontent.com/2238100/71761025-153a0300-2f1a-11ea-97af-ec095e30200c.jpg)

When I click on the menu widget to look at the other options, 'Hardware' isn't listed.
![mt2](https://user-images.githubusercontent.com/2238100/71761029-2f73e100-2f1a-11ea-9bed-071b7978b8b0.jpg)

If I click on eg. Software so the selection changes, and click again, I don't see 'Hardware' listed. The only way to get that selection back is to exit from the page without saving.
```
$ cat /data/configuration/audio_interface/alsa_controller/config.json
{
  "volumestart": {
    "type": "string",
    "value": "disabled"
  },
  "volumemax": {
    "type": "string",
    "value": "100"
  },
  "volumecurvemode": {
    "type": "string",
    "value": "logarithmic"
  },
  "outputdevice": {
    "type": "string",
    "value": "0"
  },
  "mixer_type": {
    "type": "string",
    "value": "Hardware"
  },
  "mixer": {
    "type": "string",
    "value": "PCM"
  },
  "volumesteps": {
    "type": "string",
    "value": "10"
  },
  "outputdevicename": {
    "type": "string",
    "value": "Audio Jack"
  }
}
```

**Comment by xipmix** [comment]
Strange. This seems to have gone away. I'll leave it open for now while I look for a change I made that caused it to go away.

**Comment by xipmix** [comment]
Looks like it is fixed by volumio/Volumio2/pull/1872
